ci/.gitlab-ci-cloud-experience.yaml

```diff
@@ -9,15 +9,15 @@ variables:
       - "false"
     description: "Specify whether the pipeline run is about CloudX packages testing or not."
   RUN_ON:
-    value: "aws/rhel-10.0-nightly-x86_64" # Just a defult value, needs to be updated regularly. See comments below.
+    value: "aws/rhel-10.1-nightly-x86_64" # Just a defult value, needs to be updated regularly. See comments below.
     options:
       # This list needs to be updated regularly, when new runners are introduced by osbuild-composer.
       # When a new RHEL version does GA, its nightly runner will become a "normal" runner (that is, without the "-nightly" suffix).
       # See ci/.gitlab-ci-image-builder.yaml (check "RUNNERS BLOCK").
-      - "aws/rhel-9.6-nightly-x86_64"
-      - "aws/rhel-9.6-nightly-aarch64"
-      - "aws/rhel-10.0-nightly-x86_64"
-      - "aws/rhel-10.0-nightly-aarch64"
+      - "aws/rhel-9.7-nightly-x86_64"
```

Comment:
does it mean we don't test z-streams for 9.6 and 10.0 anymore?  and similar comment further

Reply:
Comment added.

Reply:
9.6 and 10.0 are in active support, bug fixes go there, how now we can ensure that whatever goes there doesn't cause any issues, regressions, or how a fix corresponds to our test, etc?

Reply:
discussed offline, resolving for now